integration-tests/api/__tests__/store/cart/ff-tax-inclusive-pricing.js

```diff
@@ -348,4 +377,160 @@ describe("[MEDUSA_FF_TAX_INCLUSIVE_PRICING] /store/carts", () => {
       })
     })
   })
+
+  describe("POST /store/carts/:id/line-items", () => {
+    const cartIdWithItemPercentageDiscount =
+      "test-cart-w-item-percentage-discount"
+    const percentage15discountId = IdMap.getId("percentage15discountId")
+    const variantId1 = IdMap.getId("test-variant-1")
+    const variantId2 = IdMap.getId("test-variant-2")
+    const productId1 = IdMap.getId("test-product-1")
+    const productId2 = IdMap.getId("test-product-2")
+    const regionId = IdMap.getId("test-region")
+    const regionData = {
+      id: regionId,
+      includes_tax: false,
+      currency_code: "usd",
+      countries: ["us"],
+      tax_rate: 20,
+      name: "region test",
+    }
+    const buildProductData = (productId, variantId) => {
+      return {
+        id: productId,
+        variants: [
+          {
+            id: variantId,
+            prices: [],
+          },
+        ],
+      }
+    }
+    const buildPriceListData = (variantId, price, includesTax) => {
+      return {
+        status: "active",
+        type: "sale",
+        prices: [
+          {
+            variant_id: variantId,
+            amount: price,
+            currency_code: "usd",
+            region_id: regionId,
+          },
+        ],
+        includes_tax: includesTax,
+      }
+    }
+
+    describe("with a cart mixing tax inclusive and exclusive variant pricing", () => {
+      beforeEach(async () => {
+        await cartSeeder(dbConnection)
```

Comment:
nit(if-minor): we should try to move away from using the cartSeeder in favor of factories, so it is easier to understand (and build) each test in isolation.

Reply:
Yes indeed